projects/cmark/build_w_pass.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -eu
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# ! add Pass before building the fazzers
+export REPORT_FLAGS="-Xclang -load -Xclang $REPORT_PASS/libReportPass.so -flegacy-pass-manager -fPIC"
+REPORTER_FLAGS="$REPORT_PASS/libreporter.so -lc++ -pthread -lm -fPIC"
```

Comment:
why `-fPIC` is needed here?

Reply:
This project has not passed the w_pass now. One error message is below:
/usr/bin/ld: /src/cmark/ReportFunctionExecutedPass/reporter.c++.o: relocation R_X86_64_32S against `.bss' can not be used when making a shared object; recompile with -fPIC

Reply:
got it